src/aura/STG_CMP_System/STG_CMP_System.cmp

```diff
@@ -30,7 +37,7 @@
             <aura:if isTrue="{!v.isView}">
                 <ui:outputCheckbox value="{!v.hierarchySettings.Store_Errors_On__c}"/>
             <aura:set attribute="else">
-                <ui:inputCheckbox value="{!v.hierarchySettings.Store_Errors_On__c}" label="Store Erros On" labelClass="slds-hide"/>
+                <ui:inputCheckbox aura:id="storeErrors" value="{!v.hierarchySettings.Store_Errors_On__c}" label="Store Erros On" labelClass="slds-hide"/>
```

Comment:
what's the value in referencing this from the controller instead of directly like the view-based version above?